### PR TITLE
Improve test hook, fix tests (4-2-stable)

### PR DESCRIPTION
**Pull request by alanking**
Tests passed in testing environment

### CODE REVIEW

File: irods_consortium_continuous_integration_test_hook.py [hunk 1]
```diff
@@ -4,78 +4,39 @@
...
 
-    output_root_directory = options.output_root_directory
     built_packages_root_directory = options.built_packages_root_directory
     package_suffix = irods_python_ci_utilities.get_package_suffix()
     os_specific_directory = irods_python_ci_utilities.append_os_specific_directory(built_packages_root_directory)
-    irods_python_ci_utilities.install_os_packages_from_files(glob.glob(os.path.join(os_specific_directory, 'irods-rule-engine-plugin-logical-quotas*.{0}'.format(package_suffix))))
 
-    irods_python_ci_utilities.install_irods_core_dev_repository()
+    if options.do_setup:
+        # TODO: is this needed?
```
[the remark this answers is not in the document]
Comment (alanking) [reply irods_consortium_continuous_integration_test_hook.py:22]: This configures the core-dev.irods.org repository. I think this is leftover from copy-pasting out of the build hook, but I can't tell if it's needed or not, so I left it. Happy to remove if you want.

[the remark this answers is not in the document]
Comment (korydraughn) [reply irods_consortium_continuous_integration_test_hook.py:22]: You can create an issue as a reminder so we can revisit it after releasing 4.3.

If you're happy with the changes, go ahead and merge it. No need to get hung up on that.

**Review by korydraughn — APPROVED** [review]

File: irods_consortium_continuous_integration_test_hook.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (alanking) [reply irods_consortium_continuous_integration_test_hook.py:22]: Okay. I created #70 to address this.

**Comment by alanking** [comment]
I'm going to ignore the Codacy issues - the SQL is being configured as part of a test. #'d

**Pull request merged**